### PR TITLE
Scanpy plot new function

### PR DESCRIPTION
**Pull request by heylf**
FOR CONTRIBUTOR:
* [X] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [X] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

**Comment by heylf** [comment]
Second try of the PR

File: tools/scanpy/macros.xml [hunk 1]
```diff
@@ -471,6 +475,14 @@ with open('anndata_info.txt','w', encoding='utf-8') as ainfo:
         <expand macro="param_groupby"/>
         <expand macro="param_num_categories"/>
     </xml>
+    <token name="@CMD_var_names_header_check@"><![CDATA[
+header='infer'
+#if $method.var_names.type == 'customfile'
+#if $method.var_names.header == 'not_included' :
+header=None
+#end if
+#end if
```
Comment (bgruening) [inline tools/scanpy/macros.xml:484]:
```suggestion
#if $method.var_names.type == 'customfile' and $method.var_names.header == 'not_included' :
header=None
#end if
```

**Comment by heylf** [comment]
@bgruening can you just restart the github action. Seems like a connection issue.

**Review by bgruening — APPROVED** [review]
@pavanvidem ok from your side?

**Comment by nilchia** [comment]
Hi @heylf.
I did not quite understand what `@CMD_var_names_header_check@` does.
Does it only create a `header` variable?

**Comment by heylf** [comment]
@nilchia yes. As you can provide know a file to provide gene/features names that you want to plot this makro intorduces a variable to check if the file has a header or not.

**Comment by nilchia** [comment]
Aha, Now I see. Thanks a lot @heylf

**Pull request merged**